### PR TITLE
Minigames correction

### PR DESCRIPTION
**Pull request by Kiuh**

### CODE REVIEW

File: Fantasy-Quest/Assets/Scripts/Minigames/AltarGame/EditorControlPanel.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+﻿using Minigames.AltarGame.Hand;
+using UnityEngine;
+
+namespace Minigames.AltarGame
+{
+    [AddComponentMenu("Scripts/Minigames/AltarGame/Minigames.AltarGame.EditorControlPanel")]
+    internal class EditorControlPanel : MonoBehaviour
+    {
+        [SerializeField]
```
Comment (ic3mintt) [inline Fantasy-Quest/Assets/Scripts/Minigames/AltarGame/EditorControlPanel.cs:9]: [Required]

File: Fantasy-Quest/Assets/Scripts/Minigames/AltarGame/QuitInput.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+namespace Minigames.AltarGame
+{
+    [AddComponentMenu("Scripts/Minigames/AltarGame/Minigames.AltarGame.QuitInput")]
+    internal class QuitInput : MonoBehaviour
+    {
+        [Required]
+        [SerializeField]
+        private Manager manager;
+
+        [SerializeField]
```
Comment (ic3mintt) [inline Fantasy-Quest/Assets/Scripts/Minigames/AltarGame/QuitInput.cs:14]: [Required]

File: Fantasy-Quest/Assets/Scripts/Minigames/SquirrelGame/EditorControlPanel.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿using UnityEngine;
+
+namespace Minigames.SquirrelGame
+{
+    [AddComponentMenu("Scripts/Minigames/SquirrelGame/Minigames.SquirrelGame.EditorControlPanel")]
+    internal class EditorControlPanel : MonoBehaviour
+    {
+        [SerializeField]
```
Comment (ic3mintt) [inline Fantasy-Quest/Assets/Scripts/Minigames/SquirrelGame/EditorControlPanel.cs:8]: [Required]

File: Fantasy-Quest/Assets/Scripts/Minigames/SquirrelGame/Paw.cs [hunk 4]
```diff
@@ -21,9 +21,28 @@ internal class Paw : MonoBehaviour
         [SerializeField]
         private ContactFilter2D contactFilter;
 
+        [ReadOnly]
+        [SerializeField]
+        private bool inputEnabled = false;
+
         [SerializeField]
```
Comment (ic3mintt) [inline Fantasy-Quest/Assets/Scripts/Minigames/SquirrelGame/Paw.cs:28]: [Required]

File: Fantasy-Quest/Assets/Scripts/Minigames/SquirrelGame/QuitInput.cs [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+namespace Minigames.SquirrelGame
+{
+    [AddComponentMenu("Scripts/Minigames/SquirrelGame/Minigames.SquirrelGame.QuitInput")]
+    internal class QuitInput : MonoBehaviour
+    {
+        [Required]
+        [SerializeField]
+        private Manager manager;
+
```
Comment (ic3mintt) [inline Fantasy-Quest/Assets/Scripts/Minigames/SquirrelGame/QuitInput.cs:13]: [Required]

**Review by ic3mintt — APPROVED** [review]

File: Fantasy-Quest/Assets/Scripts/Minigames/SquirrelGame/Paw.cs (hunk 4 above)
Comment (Kiuh) [reply Fantasy-Quest/Assets/Scripts/Minigames/SquirrelGame/Paw.cs:28]: Struct serializing by value

File: Fantasy-Quest/Assets/Scripts/Minigames/AltarGame/QuitInput.cs (hunk 2 above)
Comment (Kiuh) [reply Fantasy-Quest/Assets/Scripts/Minigames/AltarGame/QuitInput.cs:14]: InputAction serialized by value

File: Fantasy-Quest/Assets/Scripts/Minigames/SquirrelGame/QuitInput.cs (hunk 5 above)
Comment (Kiuh) [reply Fantasy-Quest/Assets/Scripts/Minigames/SquirrelGame/QuitInput.cs:13]: InputAction serialized by value

**Pull request merged**